docs/30-getting-started.mdx

```diff
@@ -1,7 +1,9 @@
 # Getting Started
 
 1. [Install VRCFury](/download)
-1. If you're installing a VRCFury-ready prefab from an artist, stop here! Since their prefab is already setup for VRCFury, you don't need to set up your own components!
-1. Select your main avatar object, or the prop you are trying to setup, then at the bottom of the Inspector, click `Add Component` -> `VRC Fury`.
-1. Add features using the `+` button on the component. See the [Components](/components) section for information about each one.
-1. You're done! There's no "building" to do. VRCFury will non-destructively update your controllers, VRC menus and params automatically before each upload, or when you enter play mode.
+1. If you're installing a VRCFury-ready prefab from an artist, stop here! Since their prefab is already setup for VRCFury, you don't need to set up your own components! Simply drag the VRCFury-ready prefab on your avatar and follow instructions if any apply.\
+ Tip: If the asset you're trying to add to your avatar does not come with VRCFury already setup, take a look at the [Prefab sharing channel](https://discord.com/channels/1001423276553285653/1142607374096552040) on the [VRCFury Discord](https://vrcfury.com/discord), someone might already have made a VRCFury-ready community prefab for your asset!

```

Comment:
Probably don't link directly to the channel. It will 404 if they're not in the discord, and it's easy enough to find once they're in.